ArcadiaParties-SPA/src/app/app.module.ts

```diff
@@ -2,13 +2,29 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
 import { AppComponent } from './app.component';
+import { StoreModule } from '@ngrx/store';
```

Comment:
Good practice is to place all local imports after all external ones, separated with empty line.